plugin/collector/snap-plugin-collector-mock2-grpc/mock/mock_medium_test.go

```diff
@@ -96,7 +96,7 @@ func TestCollectMetric(t *testing.T) {
 
 					// only one metric for this specific hostname should be returned
 					So(len(mts), ShouldEqual, 1)
-					So(mts[0].Namespace.String(), ShouldEqual, "/intel/mock/host0/baz")
+					So(mts[0].Namespace.Strings(), ShouldResemble, plugin.NewNamespace("intel", "mock", "host0", "baz").Strings())
```

Comment:
not sure why it was like that. Probably make sure the correct library is used. Not core.Namespace